src/common-ssh/ssh.c

```diff
@@ -518,6 +520,63 @@ guac_common_ssh_session* guac_common_ssh_create_session(guac_client* client,
         return NULL;
     }
 
+    /* Check known_hosts, start by getting known_hosts file of user running guacd */
+    struct passwd *pw = getpwuid(getuid());
+    const char *known_hosts = strcat(pw->pw_dir, "/.ssh/known_hosts");
+    LIBSSH2_KNOWNHOSTS *ssh_known_hosts = libssh2_knownhost_init(session);
+    libssh2_knownhost_readfile(ssh_known_hosts, known_hosts, LIBSSH2_KNOWNHOST_FILE_OPENSSH);
+
+    /* Add host key provided from settings */
+    if (host_key && strcmp(host_key, "") > 0) {
```

Comment:
When testing for inequality, `strcmp(...) > 0` is odd. Did you mean `strcmp(...) != 0`?

Reply:
Yeah, not sure how I arrived at that.  Will change it.

Reply:
Fixed.